app/uk/gov/hmrc/gform/views/auth/enter_email.scala.html

```diff
@@ -46,11 +47,12 @@
 @govukInput = @{new govukInput(govukErrorMessage, govukHint, govukLabel)}
 @heading = @{messages("emailAuth.enterYourEmailAddress")}
 
+@* We have customised authConfig as None to hide Sign out link and disable Timeout for Email authConfig  *@
```

Comment:
We still need the timeout, with the popup. Also, is there a better way to achieve this, setting authConfig to None seems like a hack